stats/view/worker_test.go

```diff
@@ -362,9 +362,80 @@ func TestWorkerStarttime(t *testing.T) {
 	e.Unlock()
 }
 
+func TestReportUsageCommand(t *testing.T) {
+	restart()
+	ctx := context.Background()
+
+	m1 := stats.Int64("measure", "desc", "unit")
+	view1 := &View{Name: "count", Measure: m1, Aggregation: Count()}
+	m2 := stats.Int64("measure2", "desc", "unit")
+	view2 := &View{Name: "count2", Measure: m2, Aggregation: Count()}
+
+	SetReportingPeriod(5 * time.Second)
+
+	if err := Register(view1, view2); err != nil {
+		t.Fatalf("cannot register: %v", err)
+	}
+
+	e := &countExporter{}
+	RegisterExporter(e)
+
+	stats.Record(ctx, m1.M(1))
+	stats.Record(ctx, m2.M(1))
+	stats.Record(ctx, m2.M(1))
+
+	ReportUsage()
+
+	// ReportUsage should flush all views, so expect the count of 3.
+	want := int64(3)
+
+	e.Lock()
+	got := e.totalCount
+	e.Unlock()
+	if got != want {
+		t.Errorf("got count data = %v; want %v", got, want)
+	}
+}
+
+func TestUnregisterReportsUsage(t *testing.T) {
+	restart()
+	ctx := context.Background()
+
+	m1 := stats.Int64("measure", "desc", "unit")
+	view1 := &View{Name: "count", Measure: m1, Aggregation: Count()}
+	m2 := stats.Int64("measure2", "desc", "unit")
+	view2 := &View{Name: "count2", Measure: m2, Aggregation: Count()}
+
+	SetReportingPeriod(5 * time.Second)
```

Comment:
For this test, I would recommend setting the reporting period to something high to emphasize that we don't want the usual reporting cycle to occur.